Mathlib/Algebra/Module/LinearMap/End.lean

```diff
@@ -248,7 +248,7 @@ def toModuleEnd : S →* Module.End R M where
 
 end DistribMulAction
 
-namespace Module
```

Comment:
Why change this? I think as it was it matched the previous section.

Reply:
"Module" was duplicated in Module.moduleEndSelf; I'm now changing the namespace to RingEquiv to distinguish from the AlgEquiv version I'm adding.